src/Controller/isCustomConditionCheck.js

```diff
@@ -0,0 +1,58 @@
+const isSeparatorTextCheck = userInput => {
+  if (userInput[0] === '/' && userInput[1] === '/' && userInput[3] === '\\' && userInput[4] === 'n') {
+    return true;
+  }
+  return false;
+};
+
+const countMatchingAndEvenIndexChecks = userInput => {
+  let allSeparatorTrueCheck = 0;
+  let loopCountForAllSeparatorTrueCheck = 0;
+  for (let i = 6; i < userInput.length; i += 1) {
+    if (i % 2 === 0) {
+      loopCountForAllSeparatorTrueCheck += 1;
+    }
+    if (userInput[2] === userInput[i]) {
+      allSeparatorTrueCheck += 1;
+    }
+  }
+  return [allSeparatorTrueCheck, loopCountForAllSeparatorTrueCheck];
+};
+
+const isSeparatorTextInputAndLoopCountEqual = countForSeparatorTextAndLoop => {
+  if (countForSeparatorTextAndLoop[0] === countForSeparatorTextAndLoop[1]) {
+    return true;
+  }
+  return false;
+};
+
+const isSeparatorTextInputCountCheck = userInput => {
+  if (isSeparatorTextInputAndLoopCountEqual(countMatchingAndEvenIndexChecks(userInput))) {
+    return true;
+  }
+  return false;
+};
```

Comment:
이 부분은 굳이 if문으로 구분할 필요가 없을 것 같아요 

```suggestion
const isSeparatorTextInputCountCheck = userInput => {
  return isSeparatorTextInputAndLoopCountEqual(countMatchingAndEvenIndexChecks(userInput))
};
```

이렇게 해도 true/false로 표현할 수 있지 않을까요?

Reply:
여기 뿐 아니라 조건문 부분에서 조건문을 리턴해도 true/false로 구분할 수 있는 부분이 몇몇 보이는데 그 부분들에서도 반드시 if문이 필요한 것인지 고민해보면 좋을 것 같아요

Reply:
감사합니다! 이런 구조가 있는지는 몰랐네요 ㅎㅎ 🙏
if문을 생략하고 return 으로 true 값을 바로 만들 수 있는 방안을 학습해볼게요 👍